### PR TITLE
Chaged description field to (dangerous?) html

### PR DESCRIPTION
**Pull request by banzzai**
## Related Issues

- Solves: [T-27311](https://transcend.height.app/T-27311)

## Security Implications

_[none]_

## System Availability

_[none]_

### CODE REVIEW

**Review by anotherminh — COMMENTED** [review]
I don't know this well enough to review 😬 

Did you test that this works @banzzai ?

**Review by anotherminh — COMMENTED** [review]
I think you'll also need to bump the package version here, and then bump it in our monorepo, and then we'll need to run an airgap release

File: src/components/AcceptAllOrMoreChoices.tsx [hunk 1]
```diff
@@ -47,8 +47,13 @@ export function AcceptAllOrMoreChoices({
           </p>
         </div>
         <div>
-          <p className="paragraph">
-            {formatMessage(messages.acceptAllDescription)}
+        <p className="paragraph">
+            <div
+              // eslint-disable-next-line react/no-danger
+              dangerouslySetInnerHTML={{
+                __html: formatMessage(messages.acceptAllDescription),
```
Comment (blake-transcend) [inline src/components/AcceptAllOrMoreChoices.tsx:54]: Generally it's a bad idea to dangerouslySetInnerHTML for user-generated content (hence the warning). I'm not sure if we can trust our customers here tbh, so if there's a way to do this without allowing raw HTML injection I'd prefer it. I'm not sure what that would be though

**Comment by banzzai** [comment]
> I don't know this well enough to review 😬
> 
> Did you test that this works @banzzai ?

I did test, though locally... I have a few things to learn about contributing to this repo, so it seems.

File: src/components/AcceptAllOrMoreChoices.tsx (hunk 1 above)
Comment (banzzai) [reply src/components/AcceptAllOrMoreChoices.tsx:54]: Would that be something along the lines of only detecting precisely only urls and creating a href for them? If we're trusting href, that is.

Comment (banzzai) [reply src/components/AcceptAllOrMoreChoices.tsx:54]: Just playing devils advocate, I wouldn't think of contradicting the security take here, but it looks like we gave the DoNotSellExplainer a pass for this. I'm not sure if this view is much more used, that might be the concern there, and I understand that we also might be wary of starting to single case every view... I'm just trying to keep Shutterfly happy, I'm going to ask them how they would feel about leaving the url in plain text.

Comment (blake-transcend) [reply src/components/AcceptAllOrMoreChoices.tsx:54]: allowing user generated html without sanitizing it first opens the door to XSS attacks and other nefarious things. This is running on their site though, so I think it's probs less of an issue than in the AD. If they wanted to XSS themselves on their own site there's easier ways I guess. I'm not comfortable making the final OK call here but if we could get someone more familiar with this repo to look this over I'm ok with it

Comment (bencmbrook) [reply src/components/AcceptAllOrMoreChoices.tsx:54]: @banzzai mind spinning up a ticket? Would be helpful to understand the customer request motivating this change (but in a private forum / not in this open source repo). Depending on the request, there may be a safer method available.

Comment (eligrey) [reply src/components/AcceptAllOrMoreChoices.tsx:54]: Fortunately airgap can regulate the network requests made by this html. Given that this is customer provided and we have airgap as a backup, I'm fine with no sanitization

**Review by blake-transcend — DISMISSED** [review]

**Review by michaelfarrell76 — DISMISSED** [review]

**Review by michaelfarrell76 — APPROVED** [review]

**Pull request merged**